doc/rdesktop.1

```diff
@@ -123,6 +128,11 @@ but everything after this is unencrypted (including interactive logins).
 Do not send mouse motion events.  This saves bandwidth, although some Windows
 applications may rely on receiving mouse motion.
 .TP
+.BR "-M"
```

Comment:
So the main issue here is that mouse pointers doesn't work (is not shown?) when connecting to Windows 10 ?